static_inverse_kinematics.py

```diff
@@ -82,16 +81,10 @@ def __init__(
         self.q = np.zeros((self.nb_q, self.nb_frames))
         self.bounds = get_range_q(self.biorbd_model)
 
-        self.sol = []
-        self.start = None
-        self.end = None
-        self.duration = None
-        # self.residuals_xyz = [[0]*self.nb_markers*3]*self.nb_frames
-        self.residuals_xyz = np.array([[0.0] * self.nb_markers * 3] * self.nb_frames, ndmin=2)
-        self.residuals = np.array([[0.0] * self.nb_markers] * self.nb_frames, ndmin=2)
-        self.nb_iteration_diff = []
-        self.nb_iteration_jac = []
-        self.output = None
+        self.list_sol = []
+
+        self.output = dict()
+        self.nb_dim = self.xp_markers.shape[0]
```

Comment:
This should be on line 78.
Is the docstring updated?

Reply:
done